example/spec/parallel/ingest/ingestFromPdrSpec.js

```diff
@@ -69,7 +73,7 @@ const unmodifiedS3Data = [
   '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf.met',
   '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf',
 ];
-const testDataGranuleId = 'MOD09GQ.A2016358.h13v04.006.2016360104606';
+const testDataGranuleId = 'MOD09GQ.A2016358.h13v04.006.2016360104606'.replace(granuleDateString, granuleIdReplacement);
```

Comment:
Probably should explicitly CL note the PDR specs were updated to use unique granuleId.